packages/cli/package.json

```diff
@@ -28,26 +28,26 @@
     "debug": "^4.3.1",
     "extract-zip": "^2.0.1",
     "fs-extra": "^8.1.0",
+    "jimp": "^0.16.1",
     "nanoid": "^3.1.22",
     "ora": "^3.4.0",
     "pretty-bytes": "^5.6.0",
     "query-string": "^6.14.1",
-    "request": "^2.88.2",
     "semver": "^6.3.0"
   },
   "devDependencies": {
     "@types/bent": "^7.3.2",
     "@types/cli-progress": "^3.9.1",
     "@types/extract-zip": "^1.6.2",
     "@types/fs-extra": "^9.0.9",
-    "@types/request": "^2.48.5",
     "@types/semver": "^7.3.4",
     "@types/sinon": "^9.0.11",
     "ava": "^3.13.0",
     "import-fresh": "^3.3.0",
     "nyc": "^15.1.0",
     "sinon": "^10.0.0",
-    "ts-node": "^9.1.1"
+    "ts-node": "^9.1.1",
+    "typescript": "^4.3.5"
```

Comment:
Is the `typescript` required?

Reply:
I think each package should be self-contained, we need `typescript` to build ts script  to js script.